# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/windows/sdn/sdnAccessible-inl.h
# Commit: 5f5962c50e5d
# Full Hash: 5f5962c50e5d1a0d761ec320b1e22bc77c8595b4
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-07-15 21:52:49
# Description:
#   Bug 1720696: Only fetch mMsaa from non-null AccessibleWraps r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120001
# ==============================================================================

diff -r 2582c97cd813 -r 5f5962c50e5d accessible/windows/sdn/sdnAccessible-inl.h
--- a/accessible/windows/sdn/sdnAccessible-inl.h	Thu Jul 15 17:56:03 2021 +0000
+++ b/accessible/windows/sdn/sdnAccessible-inl.h	Thu Jul 15 18:23:03 2021 +0000
@@ -33,6 +33,11 @@
   // may preserve object identity.
   AccessibleWrap* wrap = static_cast<AccessibleWrap*>(
       document->GetAccessibleEvenIfNotInMap(mNode));
+
+  if (!wrap) {
+    return nullptr;
+  }
+
   wrap->GetNativeInterface(getter_AddRefs(mMsaa));
   return mMsaa;
 }
